src/core/js/startController.js

```diff
@@ -13,7 +13,16 @@ define([
         setStartLocation: function() {
             if (!this.isEnabled()) return;
 
-            window.location.hash = this.getStartHash();
+            var hash = this.getStartHash();
+
+            if ('replaceState' in window.history) {
+                window.history.replaceState('', '', hash);
+            } else {
+                // < ie9
+                if (window.location.hash !== hash) window.history.back();
```

Comment:
my only comment would be that this line needs a bit more explanation than just '< IE9'